test/powershell/Language/Parser/TypeAccelerator.Tests.ps1

```diff
@@ -1,17 +1,411 @@
+            }
+            @{
+                Accelerator = 'type'
+                Type        = [System.Type]
+            }
+            @{
+                Accelerator = 'psmoduleinfo'
+                Type        = [System.Management.Automation.PSModuleInfo]
+            }
+            @{
+                Accelerator = 'powershell'
+                Type        = [System.Management.Automation.Powershell]
+            }
+            @{
+                Accelerator = 'runspacefactory'
+                Type        = [System.Management.Automation.Runspaces.RunspaceFactory]
+            }
+            @{
+                Accelerator = 'runspace'
+                Type        = [System.Management.Automation.Runspaces.Runspace]
+            }
+            @{
+                Accelerator = 'initialsessionstate'
+                Type        = [System.Management.Automation.Runspaces.InitialSessionState]
+            }
+            @{
+                Accelerator = 'psscriptmethod'
+                Type        = [System.Management.Automation.PSScriptMethod]
+            }  
+            @{
+                Accelerator = 'psscriptproperty'
+                Type        = [System.Management.Automation.PSScriptProperty]
+            }
+            @{
+                Accelerator = 'psnoteproperty'
+                Type        = [System.Management.Automation.PSNoteProperty]
+            }
+            @{
+                Accelerator = 'psaliasproperty'
+                Type        = [System.Management.Automation.PSAliasProperty]
+            }
+            @{
+                Accelerator = 'psvariableproperty'
+                Type        = [System.Management.Automation.PSVariableProperty]
+            }
+        )
+
+        if (!$IsCoreCLR)
+        {
+            $nonCoreTypeAcceleratorTestCases = @(
+                @{
+                    Accelerator = 'adsi'
+                    Type        = [System.DirectoryServices.DirectoryEntry]
+                }
+                @{
+                    Accelerator = 'adsisearcher'
+                    Type        = [System.DirectoryServices.DirectorySearcher]
+                }
+                @{
+                    Accelerator = 'wmiclass'
+                    Type        = [System.Management.ManagementClass]
+                }
+                @{
+                    Accelerator = 'wmi'
+                    Type        = [System.Management.ManagementObject]
+                }
+                @{
+                    Accelerator = 'wmisearcher'
+                    Type        = [System.Management.ManagementObjectSearcher]
+                }
+            )
+        }
+    }
 
     It "Basic type accelerator usage" {
         [msft_2174855] | Should Be ([int])
     }
 
     It "Can query type accelerators" {
-        if ( $IsCoreCLR ) { $count = 80 } else { $count = 82 }
+        if ( $IsCoreCLR )
+        {
+            $count = 80 
+        } else
+        {
+            $count = 82 
+        }

```

Comment:
I believe it is better use the pattern and formatting https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/DefaultCommands.Tests.ps1